src/coreclr/gc/gc.cpp

```diff
@@ -48376,6 +48376,17 @@ enable_no_gc_region_callback_status GCHeap::EnableNoGCRegionCallback(NoGCRegionC
     return gc_heap::enable_no_gc_callback(callback, callback_threshold);
 }
 
+uint64_t GCHeap::GetGenerationBudget(int generation)
```

Comment:
could you please move this into gcee.cpp? that's where we exposed all the API for diagnostics stuff